CMakeLists.txt

```diff
@@ -25,8 +25,11 @@ FetchContent_Declare(asio
     GIT_TAG f693a3eb7fe72a5f19b975289afc4f437d373d9c)
 FetchContent_Declare(llhttp
     URL "https://github.com/nodejs/llhttp/archive/refs/tags/release/v8.1.0.tar.gz")
+FetchContent_Declare(fmt
+    GIT_REPOSITORY https://github.com/fmtlib/fmt.git
+    GIT_TAG a0b8a92e3d1532361c2f7feb63babc5c18d00ef2)
```

Comment:
Adding a new OSS dependency requires some more work. We should chat offline.

Reply:
I'm not sure fmt is needed here in the first place actually. only `fmt::sprintf` is used. These 2 calls can be replaced with ìtoa` and `ftoa` or `sprintf`

Reply:
If that works, it is much better than adding a new dependency.

Reply:
using sprintf_s instead 😄

Reply:
We have to be very careful using printf-like functions with a user supplied format string. There are lots of possible security issues here.